myScript.js

```diff
@@ -0,0 +1,72 @@
+//Task 1 --------------------------------------------------------------------------------------------
+{
+    const userNames = ["Петрик Ольга Іванівна", "Гнатюк Петро Антонович", "Рудко Андрій Опанасович"];
+    let initials = [];
+    
+    for (let fullName of userNames)
+    {
+        let name = fullName.split(' ');  
+        initials.push(name.map((x)=>x[0]).join('.'));
+    }
+    initials.sort();
+
+    console.log(initials); // [ "Г.П.А.", "П.О.І.", "Р.А.О."]
+}
+//---------------------------------------------------------------------------------------------------
+
+
+//Task 2.1 ------------------------------------------------------------------------------------------
+{
+    const userNames = ['Петро', 'Емма', 'Юстин', 'Ілля', 'Марта', 'Яна', 'Василь', 'Антон', 'Олена'];
+    let filteredNames=[];
+    const letters = ['А','Е','О','У','Є','Ю','І','Ї','Я'];
+
+    for (let fName of userNames)
+    {
+        if(letters.includes(fName[0]))
+        {
+            filteredNames.push(fName);
+        }
+    }
```

Comment:
```suggestion
    for (let fName of userNames) {
        if(letters.includes(fName[0])) {
            filteredNames.push(fName);
        }
    }
```
Зробить більш загальноприйняте форматування)